### PR TITLE
fix: update flair encoder

### PR DESCRIPTION
**Pull request by Yongxuanzhang**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM jinaai/jina:2.0.0rc6
+FROM jinaai/jina:2.0.0rc9
```
Comment (JoanFM) [inline Dockerfile]: rc10 better

File: requirements.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-jina==2.0.0rc6
+jina==2.0.0rc9
```
Comment (JoanFM) [inline requirements.txt]: 10 bettee

**Review by JoanFM — DISMISSED** [review]

File: Dockerfile (hunk 1 above)
Comment (Yongxuanzhang) [reply Dockerfile]: no problem

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**